index.js

```diff
@@ -43,7 +45,7 @@ class IsConnected extends EventEmitter {
          */
     _dns() {
             if (os.platform() === 'linux') {
-                setInterval(function() {
+                setTimeout(function() {
```

Comment:
Hi, 
i like the ping timeout change, and for usage of setInterval it is because i wanted this to always be running and give me a event of disconnection as soon as it does, setTimeout will defeat that purpose as it will only run once when the timer times-out. Maybe there is a better implementation? that i missed out what do you think?